Index: javaSRC/markov_databse/verse_db/src/Database.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- javaSRC/markov_databse/verse_db/src/Database.java	(revision 4cc6f3beb3e47ed586a0065fd67c75f80e043086)
+++ javaSRC/markov_databse/verse_db/src/Database.java	(revision )
@@ -85,7 +85,7 @@
                             word = stripWord(word);
 
                             if(word.length() == 1) { //removes all single character words that aren't 'a' or 'i'
-                                if(!word.equals("a") || !word.equals("i")) {
+                                if(!word.equals("a") && !word.equals("i")) {
                                     continue;
                                 }
                             }
@@ -148,7 +148,7 @@
         if(wordMap.containsKey(word)) {
             return wordMap.get(word);
         } else {
-            System.out.println("WORD NOT IN wordMap DATABASE");
+            System.out.println(word+" NOT IN wordMap DATABASE");
             return null;
         }
     }
@@ -173,8 +173,8 @@
 
     public static void main(String [] args) {
         String path = "C:\\Users\\Dawn\\Documents\\GitHub\\VerseArchive\\allpoems.txt";
-        MakeVerse verse = new MakeVerse("");
-        System.out.println(verse.sortedNeighbors("of"));
+        MakeVerse verse = new MakeVerse("Love");
+        System.out.println(verse);
 
 
 
\ No newline at end of file
Index: .idea/RESTGET.java
===================================================================
--- .idea/RESTGET.java	(revision 4cc6f3beb3e47ed586a0065fd67c75f80e043086)
+++ .idea/RESTGET.java	(revision 4cc6f3beb3e47ed586a0065fd67c75f80e043086)
@@ -1,42 +0,0 @@
-
-
-import java.io.IOException;
-import javax.servlet.ServletException;
-import javax.servlet.annotation.WebServlet;
-import javax.servlet.http.HttpServlet;
-import javax.servlet.http.HttpServletRequest;
-import javax.servlet.http.HttpServletResponse;
-
-/**
- * @author Andrew Berson (axb4069)
- * Servlet implementation class RESTGET
- */
-@WebServlet("/RESTGET")
-public class RESTGET extends HttpServlet {
-	private static final long serialVersionUID = 1L;
-       
-    /**
-     * @see HttpServlet#HttpServlet()
-     */
-    public RESTGET() {
-        super();
-        // TODO Auto-generated constructor stub
-    }
-
-	/**
-	 * @see HttpServlet#doGet(HttpServletRequest request, HttpServletResponse response)
-	 */
-	protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
-		response.setContentType("text/plain");
-		
-		response.getWriter().write(new MakeVerse(request.));
-	}
-
-	/**
-	 * @see HttpServlet#doPost(HttpServletRequest request, HttpServletResponse response)
-	 */
-	protected void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
-		// TODO Auto-generated method stub
-	}
-
-}
